.travis.yml

```diff
@@ -38,6 +38,8 @@ matrix:
     env: KAFKA_VERSION=1.0.1
   - scala: 2.11
     env: KAFKA_VERSION=1.1.0
+  - scala: 2.11
```

Comment:
This should just replace `1.1.0`. `1.1.1` is the latest version of the `1.1.x branch`.